### ISSUE TITLE
kubernetes 1.0.3 kube-up.sh  not functioning

### ISSUE THREAD

**Issue by hiltyj** [kubernetes/kubernetes#12877]
When I attempt to kube-up it gets most of the way completed but then stops and gives
```
./../cluster/libvirt-coreos/../../cluster/common.sh: line 83: user_args[@]: unbound variable
```
Is this just me or has this occurred elsewhere?

**Comment by roberthbailey** [comment]
@lhuard1A

**Comment by rubenv** [comment]
This is fixed in master, but sadly not released yet: https://github.com/kubernetes/kubernetes/blob/b5a4a548df0cffb99bdcc3b9b9e48d4025d0541c/cluster/common.sh#L83-L85

**Comment by roberthbailey** [comment]
@rubenv Is there a PR that should be cherry picked into the 1.0 release branch to fix this when we cut 1.0.4?

**Comment by rubenv** [comment]
@roberthbailey Looks like #11794, but that depends on other things. I'm not up-to-date on all the details, maybe best to ping @mikedanese who wrote it.
